docs/sources/tempo/configuration/_index.md

```diff
@@ -1083,6 +1083,17 @@ storage:
             #  create larger footers but will be harder to shard when searching. It is difficult to calculate
             #  this field directly and it may vary based on workload. This is roughly a lower bound.
             [parquet_row_group_size_bytes: <int> | default = 100MB]
+
+            # *vParquet3-only configuration*
+            # Configures attributes to be stored as dedicated columns in the parquet file, rather than in the
+            # generic attribute key-value list. This allows for more efficient searching of these attributes.
+            # Up to 10 span attributes and 10 resource attributes can be configured as dedicated columns.
```

Comment:
Maybe present the vParquet3 requirement a bit differently. We don't make use of this `*..*` notation in other parts of the configuration documentation:
```suggestion
            # Configures attributes to be stored as dedicated columns in the parquet file, rather than in the
            # generic attribute key-value list. This allows for more efficient searching of these attributes.
            # Up to 10 span attributes and 10 resource attributes can be configured as dedicated columns.
            # Requires vParquet3
```